ee/ucp/kubernetes/storage/use-csi.md

```diff
@@ -15,22 +15,21 @@ Docker Enterprise 3.0 supports version 1.0+ of the CSI specification. Therefore,
 
 ![Kubernetes and CSI components](/ee/ucp/images/csi-plugins.png){: .with-border}
 
-**Note**: Docker Enterprise does not provide CSI drivers. CSI drivers are provided by enterprise storage vendors. 
-Kubernetes does not enforce a specific procedure for how Storage Providers (SP) should bundle and distribute CSI drivers.
+> Note
+>
+> Docker Enterprise does not provide CSI drivers. CSI drivers are provided by enterprise storage vendors. Kubernetes does not enforce a specific procedure for how Storage Providers (SP) should bundle and distribute CSI drivers.
 
 Review the [Kubernetes CSI Developer Documentation](https://kubernetes-csi.github.io/docs/) for CSI architecture, 
 security, and deployment details.
 
 ## Prerequisites
 
-1. Select a storage provider from the list of [available CSI drivers](https://kubernetes-csi.github.io/docs/drivers.html) 
-or as documented by your storage vendor. 
+1. Select a CSI driver to use with Kubernetes. Contact Docker for a list of certified drivers.
```

Comment:
Contacting docker everytime a customer wants to know about the list of certified drivers is very cumbersome. Why dont we list the drivers per UCP release in this doc or create another link and reference that here?

Reply:
@anusha-ragunathan we can certainly do that on CSI page. We could remove the K8s community CSI drivers from the current page and add table with list of certified drivers.
@khudgins

Reply:
great. lets do that please.